### PR TITLE
Updates on main OCR Flow commands

### PR DESCRIPTION
**Pull request by RodrigoAD**

### CODE REVIEW

**Comment by RodrigoAD** [comment]
> I think you might be missing the `set_writer` command in the flow command?

Ok, adding it now

File: gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/initialize.flow.ts [hunk 1]
```diff
@@ -35,8 +50,15 @@ export default class OCR2InitializeFlow extends FlowCommand<TransactionResponse>
         id: this.stepIds.OCR_2,
       },
       {
-        name: 'Change RDD',
-        exec: this.showRddInstructions,
+        name: 'Change RDD Identifier',
+        exec: this.showRddIdentifierInstructions,
+      },
+      {
+        name: 'Set writer on Store',
```
Comment (RodrigoAD) [inline gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/initialize.flow.ts:57]: @aalu1418 This is run right after the OCR2 feed is initialized. I think is correct?

Comment (aalu1418) [reply gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/initialize.flow.ts:57]: I think that should work. all you need is the store account (from the initialize store), transmissions account (from the create feed), and storeAuthority (calculated in the ocr2 initialize)

File: gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/store/setWriter.ts [hunk 2]
```diff
@@ -1,15 +1,22 @@
 import { Result } from '@chainlink/gauntlet-core'
-import { logger, BN } from '@chainlink/gauntlet-core/dist/utils'
+import { logger } from '@chainlink/gauntlet-core/dist/utils'
 import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
 import { PublicKey } from '@solana/web3.js'
+import { utils } from '@project-serum/anchor'
 import { CONTRACT_LIST, getContract } from '../../../lib/contracts'
+import { getRDD } from '../../../lib/rdd'
+
+type Input = {
```
Comment (RodrigoAD) [inline gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/store/setWriter.ts:9]: Changed a bit the command interface to be more understandable. User should provide the ocr2 state address as flag. If RDD is provided, transmissions and store should be there. 
`storeAuthority` is calculated on the fly

File: gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/store/createFeed.ts [hunk 3]
```diff
@@ -49,24 +47,21 @@ export default class CreateFeed extends SolanaCommand {
...
 
     const store = new PublicKey(input.store)
-    const state = new PublicKey(this.flags.state)
     const feed = Keypair.generate()
 
     const granularity = new BN(input.granularity)
     const liveLength = new BN(input.liveLength)
     const length = new BN(this.flags.length || 8096)
-    const feedAccountLength = new BN(8 + 128 + length.toNumber() * 24)
+    const feedAccountLength = new BN(8 + 128 + length.toNumber() * 48)
```
Comment (aalu1418) [inline gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/store/createFeed.ts:55]: Shouldn't this still be `24`? the transmissions account didn't change

**Review by aalu1418 — CHANGES_REQUESTED** [review]

**Review by aalu1418 — APPROVED** [review]
👍🏼

**Pull request merged**